keps/sig-apps/1847-autoremove-statefulset-pvcs/README.md

```diff
@@ -407,72 +416,71 @@ are not involved so there is no version skew between nodes and the control plane
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine 
 the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+  - Metric name: `statefulset_reconcile_delay`
+    - [Optional] Aggregation method: `quantile`
+    - Components exposing the metric: `pke/controller/statefulset`
+  - Metric name: `statefulset_unhealthy_pods`
+    - [Optional] Aggregation method: `sum`
+    - Components exposing the metric: `pke/controller/statefulset`
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  At a high level, this usually will be in the form of "high percentile of SLI
-  per day <= X". It's impossible to provide comprehensive guidance, but at the very
-  high level (needs more precise definitions) those may be things like:
-  - per-day percentage of API calls finishing with 5XX errors <= 1%
-  - 99% percentile over day of absolute value from (job creation time minus expected
-    job creation time) for cron job <= 10%
-  - 99,9% of /health requests per day finish with 200 code
+
+  The reconcile delay (time between statefulset reconcilliation loops) should be
+  low. For example, the 99%ile should be at most minutes.
+  
+  This can be combined with the unhealthy pod count, although as unhealthy pods
+  are usually an application error rather than a problem with the stateful set
+  controller, this will be more a decision for the operator to decide on a
+  per-cluster basis.
 
 * **Are there any missing metrics that would be useful to have to improve observability 
 of this feature?**
-  Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
-  implementation difficulties, etc.).
 
-### Dependencies
+  The stateful set controller has not had any metrics in the past despite it
+  being a core Kubernetes feature for some time. Hence which metrics are useful
+  in practice is an open question in spite of the stability of the feature.
 
-_TBD upon graduation to beta._
+### Dependencies
 
 * **Does this feature depend on any specific services running in the cluster?**
-  Think about both cluster-level services (e.g. metrics-server) as well
-  as node-level agents (e.g. specific version of CRI). Focus on external or
-  optional services that are needed. For example, if this feature depends on
-  a cloud provider API, or upon an external software-defined storage or network
-  control plane.
-
-  For each of these, fill in the following—thinking about running existing user workloads
-  and creating new ones, as well as about cluster-level services (e.g. DNS):
-  - [Dependency name]
-    - Usage description:
-      - Impact of its outage on the feature:
-      - Impact of its degraded performance or high-error rates on the feature:
+
+  No, outside of dependin on the scheduler, the garbage collector and volume
```

Comment:
s/dependin/depending/

Reply:
troubleshooting done, and typo fixed, ptal

Thanks!